enterprise_subsidy/apps/transaction/management/commands/backpopulate_transaction_course_run_start_date.py

```diff
@@ -0,0 +1,55 @@
+import logging
+from django.core.management.base import BaseCommand
+from openedx_ledger.models import Transaction
+from enterprise_subsidy.apps.content_metadata.api import ContentMetadataApi
+from enterprise_subsidy.apps.subsidy.models import Subsidy
+from dateutil import parser
+from enterprise_subsidy.apps.transaction.utils import normalize_to_datetime
+
+logger = logging.getLogger(__name__)
+
+class Command(BaseCommand):
+    help = "Backpopulating course_run_start_date for Transactions by fetching from content metadata."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--since',
+            type=str,
+                help='Only update transactions modified on or after this date (ISO format, e.g. 2025-09-01)',
+            required=False,
+        )
+
+    def handle(self, *args, **options):
+        content_api = ContentMetadataApi()
+        transactions = Transaction.objects.filter(course_run_start_date__isnull=True, content_key__isnull=False).exclude(content_key='')
+        since = options.get('since')
+        if since:
+            try:
+                since_date = normalize_to_datetime(since)
+                transactions = transactions.filter(modified__gte=since_date)
+                logger.info(f"Filtering transactions modified since {since_date}")
+            except Exception as e:
+                logger.error(f"Invalid date for --since: {since}. Error: {e}")
+                return
+        logger.info(f"Starting backpopulate for {transactions.count()} transactions")
+        updated_count = 0
+        error_count = 0
+        for transaction in transactions.iterator(chunk_size=100):
+            try:
+                subsidy = getattr(transaction.ledger, 'subsidy', None)
+                if transaction.ledger and subsidy:
+                    enterprise_customer_uuid = subsidy.enterprise_customer_uuid
+                    content_summary = content_api.get_content_summary(enterprise_customer_uuid, transaction.content_key)
+                    course_run_start_date_str = content_summary.get('course_run_start_date')
```

Comment:
[question] We are passing a content_key here but the "start" date lives in course run. Have we verified what does the content_api.get_content_summary return us against a course key? Do we need to update the statement to pass the course run id instead?